test/verify/check-connection

```diff
@@ -966,6 +966,51 @@ until pgrep -f '^/usr/bin/cockpit-bridge.*--privileged'; do sleep 1; done
         m.write("/etc/cockpit/ws-certs.d/0-self-signed-ca.pem", "FAKE CERT FOR TESTING\n")
         self.assertEqual(m.execute("curl -sfS http://localhost:9090/ca.cer"), "FAKE CERT FOR TESTING\n")
 
+    def test_branding(self):
+        m = self.machine
+
+        # for all of our CI images, the part before the dash is the name of the
+        # subdirectory in /usr/share/cockpit/branding.
+        brand = m.image.split('-')[0]
+        branddir = f'/usr/share/cockpit/branding/{brand}'
+
+        # make sure that there are no broken links in "our" directory
+        self.assertFalse(m.execute(f'find {branddir} -xtype l'))
```

Comment:
Please append a `.strip()` and use `assertEqual` with `""`, so that we get useful assertion failures.

Reply:
i originally had assertEqual.  Will put it back :)